File: PHP/UberBlack.php
<?php
require_once('uber_cars.php');
class UberBlack extends uber_cars {
    public $typeCarAccepted;
    public $material;

    public function __construct($license, $driver, $typeCarAccepted, $material) {
        parent:: __construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->material = $material;
    }
}
?>

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Uber_cars {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> material;

    public UberBlack(String license, Cuenta driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> material){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.material = material;
    }
}
